tasks/add_additional_nrpe_checks.yml

```diff
@@ -9,21 +9,22 @@
   with_items: "{{ additional_nrpe_checks }}"
   when: item.type == 'git'
 
-- name: Set default nrpe command argument when nagios_dont_blame is true
-  set_fact:
-    default_argument: "$ARG1$"
-  when: nagios_dont_blame == true
-- name: Set default nrpe command argument when nagios_dont_blame is false
-  set_fact:
-    default_argument: ""
-  when: nagios_dont_blame == false
+- name: set SELinux contexts for checks
+  sefcontext:
+    target: "{{ item[0].script_path + item[1].script_name }}"
+    setype: "{{ item[1].setype }}"
```

Comment:
Could you provide an example of setype in readme and/or defaults

Reply:
added.